src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java

```diff
@@ -82,13 +82,11 @@ public void onServerListPing(ServerListPingEvent event) {
 
   @EventHandler(priority = EventPriority.HIGHEST)
   public void onJoin(final PlayerJoinEvent event) {
-    event.setJoinMessage("");
```

Comment:
Why did you remove that, join and quit should be silent for bungee, you don't see messages "xyz joined the game" on big networks.

Reply:
This I think is superfluous unnecessary, as there are plugins that handle join & quit messages.

Reply:
You think it's unnecessary while someone might see this feature as something good. Minigame is meant to run by its own without installing additional plugins.

Reply:
Rather, I leave this feature, may be another user.